### PR TITLE
Security Process: Add Distributors List Process

### PR DESCRIPTION
**Pull request by jessfraz**
cc @philips @cjcullen @tallclair @liggitt 

First draft based off items in http://oss-security.openwall.org/wiki/mailing-lists/distros

### CODE REVIEW

File: contributors/devel/security-release-process.md [hunk 1]
```diff
@@ -98,6 +99,70 @@ These steps should be completed 1-3 days after the Release Date. The retrospecti
...
+   issues.
+4. Not be a downstream or rebuild of another distribution.
+5. Be a participant and active contributor in the community.
+6. Accept the [Embargo Policy](#embargo-policy) that is outlined above.
+7. Be willing to [contribute back](#contributing-back) as outlined above.
+8. Be willing and able to handle PGP-encrypted email.
+9. Have someone already on the list vouch for the person requesting membership
+   on behalf of your distribution.
+
+New membership requests are discussed in public on {TODO}.
```
Comment (jessfraz) [inline contributors/devel/security-release-process.md]: Unsure where we wanted this to happen? Or should we just have them ask on security@

**Comment by jessfraz** [comment]
I love how I found all the typos after I opened the PR 🤦‍♀️  lol

**Comment by philips** [comment]
@jessfraz missing the steps to request to join.

**Comment by jessfraz** [comment]
Okay I added an example of a request... I figure after we have a few we can make it a bit more of a real example, but I obviously had some fun with it

**Comment by jessfraz** [comment]
I could always make it serious as well 😇

**Comment by philips** [comment]
@jessfraz 👍

**Pull request merged**